### ISSUE TITLE
Automated docker builds failing (bundler error?)

### ISSUE THREAD

**Issue by daturkel** [cantino/huginn#1232]
The [automated docker builds](https://hub.docker.com/r/cantino/huginn/builds/) have been failing for the past few days. I was trying to build a container with minor tweaks in [my fork](https://github.com/daturkel/huginn/commit/77795917550ac49a604028d35bb522cba8016e2f) (you can check and see that my tiny changes are not likely to have broken anything) and I noticed the error in the build process: 
```bash
$ docker build --rm=true --tag=daturkel/huginn .
Sending build context to Docker daemon 40.45 kB
[snip]
Step 9 : RUN /scripts/setup
 ---> Running in 99bf83fd8bcc
Cloning into '/app'...
Adding group `huginn' (GID 1000) ...
Done.
Adding user `huginn' ...
Adding new user `huginn' (1000) with group `huginn' ...
Not creating home directory `/app'.
Adding user `huginn' to group `sudo' ...
Adding user huginn to group sudo
Done.
passwd: password expiry information changed.
Fetching gem metadata from https://rubygems.org/.......
Fetching version metadata from https://rubygems.org/...
Fetching dependency metadata from https://rubygems.org/..
Fetching git://github.com/collectiveidea/delayed_job_active_record.git
/var/lib/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/ui/shell.rb:98:in `[]': invalid byte sequence in US-ASCII (ArgumentError)
	from /var/lib/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/ui/shell.rb:98:in `strip_leading_spaces'
	from /var/lib/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/ui/shell.rb:103:in `word_wrap'
	from /var/lib/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/ui/shell.rb:89:in `tell_me'
	from /var/lib/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/ui/shell.rb:34:in `error'
	from /var/lib/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/friendly_errors.rb:20:in `rescue in with_friendly_errors'
	from /var/lib/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/friendly_errors.rb:7:in `with_friendly_errors'
	from /var/lib/gems/2.1.0/gems/bundler-1.11.2/exe/bundle:17:in `<top (required)>'
	from /usr/local/bin/bundle:23:in `load'
	from /usr/local/bin/bundle:23:in `<main>'
The command '/bin/sh -c /scripts/setup' returned a non-zero code: 1
```
Is this some sort of error with bundler itself? I can confirm I have the latest version of the bundler gem.

**Comment by daturkel** [comment]
Ah, this might be fixed by adding `export LANG=C.UTF-8` to the Dockerfile as per http://stackoverflow.com/questions/33559075/how-to-run-bundle-install-with-a-dockerfile-file

**Comment by dsander** [comment]
Hi @daturkel,

thanks for noticing the failed build! I probably should set up a notification about that.

Setting `LANG` almost did it, we also need need to run `locale-gen`. I updated my pull request #1204 with a fix. Do you mind testing the `dsander/huginn` image? It should behave exactly as `cantino/huginn` does/did before the build failed.

**Comment by daturkel** [comment]
Thanks for the quick fix! Will test your fork tonight.

**Comment by kennethkalmer** [comment]
Fixed since #1204 got merged

**Comment by waldens** [comment]
Also get this error when follow [this update manual](https://github.com/cantino/huginn/blob/master/doc/manual/update.md), at step 4: run `bundle install`.
and so thanks @dsander , `locale-gen` is working and fix it!
